mock/py/mock.py

```diff
@@ -529,17 +529,17 @@ def check_arch_combination(target_arch, config_opts):
 
     config.multiply_platform_multiplier(config_opts)
     if config_opts['forcearch']:
-        binary = '/usr/bin/qemu-x86_64-static'
+        binary = '/usr/bin/qemu-{}-static'.format(config_opts['forcearch'])
```

Comment:
This will not work for most architectures, as the binary is named by the architecture family name rather than the actual architecture name.

For example, ARM architectures are in `qemu-arm-static`, POWER architectures are in `qemu-ppc-static`, and so on.

Reply:
Damm it. I swear I checked... Thanks, Neal for noticing.

So I see two options for how to handle it. Either translate it in mock code. Or introduce a new config and command line option.  Opinions?

Reply:
Probably an architecture map for qemu-static binaries should be introduced in the mock core config/code.

Reply:
Mapping added.